dist/tablesaw.js

```diff
@@ -372,13 +374,26 @@ if( Tablesaw.mustard ) {
 
 		this.$table.addClass( this.classes.columnToggleTable );
 
+		$btnContain = $( document.createElement( "div" ) ).addClass( this.classes.columnBtnContain );
+
 		tableId = this.$table.attr( "id" );
-		id = tableId + "-popup";
-		$btnContain = $( "<div class='" + this.classes.columnBtnContain + "'></div>" );
-		$menuButton = $( "<a href='#" + id + "' class='btn btn-micro " + this.classes.columnBtn +"' data-popup-link>" +
-										"<span>" + Tablesaw.i18n.columnBtnText + "</span></a>" );
-		$popup = $( "<div class='dialog-table-coltoggle " + this.classes.popup + "' id='" + id + "'></div>" );
-		$menu = $( "<div class='btn-group'></div>" );
+		popupId = tableId + "-popup";
+
+		$menuButton = $( "<a>", {
+			id: popupId,
+			class: "btn btn-micro " + this.classes.columnBtn
```

Comment:
Be careful, [`class` needs to be quoted here](http://api.jquery.com/jQuery/#creating-new-elements):

> The name "class" must be quoted in the object since it is a JavaScript reserved word, and "className" cannot be used since it refers to the DOM property, not the attribute.

You can also move the `data-popup-link` attribute from below up here as long as it's quoted as well.